gen/armor.py

```diff
@@ -545,6 +546,14 @@ def plan_action_for_groups(
                 dcs_group.add_waypoint(retreat_point, PointAction.OffRoad)
                 dcs_group.add_waypoint(reposition_point, PointAction.OffRoad)
 
+            if len(dcs_group.points) >= 2:
+                try:
+                    dcs_group.points[1].ETA = from_cp.delayed_start.total_seconds()
+                    dcs_group.points[1].ETA_locked = True
+                    dcs_group.points[1].speed_locked = False
+                except AttributeError:  # old saves may not have delayed_start
+                    pass
```

Comment:
A better way to do this is with `__setstate__`. There are examples of how to do it elsewhere in the codebase.